src/components/LiveCursors/CursorSvg.tsx

```diff
@@ -0,0 +1,39 @@
+const CursorSvg = ({
+  startColor,
+  endColor,
+  id,
+}: {
+  startColor: string
+  endColor: string
+  id: string | undefined
```

Comment:
This should always be a `string` (and code using this component should ensure that) otherwise it will produce broken components (with `id` of `gradient-undefined`)